mobilitydb/test/general/expected/022_temporal.test.out

```diff
@@ -6717,19 +6717,19 @@ SELECT minusValues(tfloat '[1@2000-01-01, 2@2000-01-02]', 1 - 1e-17);
 SELECT minusValues(tfloat '[1@2000-01-01, 2@2000-01-02]', 1 + 1e-12);
                             minusvalues                             
 --------------------------------------------------------------------
- {(1@Sat Jan 01 00:00:00 2000 PST, 2@Sun Jan 02 00:00:00 2000 PST]}
+ {[1@Sat Jan 01 00:00:00 2000 PST, 2@Sun Jan 02 00:00:00 2000 PST]}
 (1 row)
 
 SELECT minusValues(tfloat '(1@2000-01-01, 2@2000-01-02]', 1 + 1e-12);
- minusvalues 
--------------
- 
+                            minusvalues                             
+--------------------------------------------------------------------
+ {(1@Sat Jan 01 00:00:00 2000 PST, 2@Sun Jan 02 00:00:00 2000 PST]}
 (1 row)
```

Comment:
Is it normal that this changed? The second query seems correct, which it wasn't before. But I don't understand why the answer to the first query changed? I think the old answer (`{(1@Sat Jan 01 00:00:00 2000 PST, 2@Sun Jan 02 00:00:00 2000 PST]}`) was correct.

Reply:
The condition for locating a value in a segment were uniformized in all *segm_locate() functions 
https://github.com/estebanzimanyi/MobilityDB/blob/tcbuffer/meos/src/general/tsequence.c#L232